.claude/requirements/todo/REQ-007.md

```
@@ -0,0 +1,21 @@
+# [REQ-007] 축하하기
+
+## 개요
+친구 활동 피드에서 축하하기 버튼을 눌러 해당 유저에게 LP를 지급하고 알림을 전송한다.
+
+## API
+| Method | Path | 설명 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)